apps/src/StudioApp.js

```diff
@@ -1217,6 +1232,19 @@ StudioApp.prototype.setCheckForEmptyBlocks = function (checkBlocks) {
 };
 
 /**
+ * Add the starting block(s).
+ */
+StudioApp.prototype.setStartBlocks_ = function (config) {
+  var startBlocks = config.level.startBlocks || '';
+  if (config.forceInsertTopBlock) {
+    startBlocks =
+        blockUtils.forceInsertTopBlock(startBlocks, config.forceInsertTopBlock);
```

Comment:
I personally prefer something like
```
startBlocks = blockUtils.forceInsertTopBlock(startBlocks,
    config.forceInsertTopBlock)
```
but it does look like style guide says how you have it is okay too.